package-testing/spec/package/update_module_spec.rb

```diff
@@ -3,15 +3,13 @@
 require 'json'
 
 modules = [
-  'puppetlabs/puppetlabs-motd',
-  'puppetlabs/puppetlabs-concat',
-  'puppetlabs/puppetlabs-inifile'
+  'puppetlabs/puppetlabs-motd'
```

Comment:
what's the rationale behind only testing against motd?

Reply:
It reduces the surface of issues. MOTD changes are less frequent than the others.